compositor/view.c

```diff
@@ -93,9 +93,13 @@ void wc_view_damage(struct wc_view *view, pixman_region32_t *damage) {
 
 	for (size_t i = 0; i < 4; i++) {
 		struct wlr_output *output = outputs[i];
+		struct wlr_box *output_box =
+				wlr_output_layout_get_box(view->server->output_layout, output);
+
 		if (output) {
-			struct wlr_box view_output_geo = {.x = view->geo.x - output->lx,
-					.y = view->geo.y - output->ly,
+			// TODO replace lx and ly with xdg-output
+			struct wlr_box view_output_geo = {.x = view->geo.x - output_box->x,
+					.y = view->geo.y - output_box->y,
 					.width = view->geo.width,
 					.height = view->geo.height};
```

Comment:
Also while you're at it, can you add a trailing comma here after `view->geo.height`? clang-format has a bug (at least I hope it's a bug) where it does ugly formatting of this inline struct definition unless you add a trailing comma.